DI-1299/README.md

```
@@ -0,0 +1,53 @@
+
+### Running
+Example command:
+```
+python compare_b37_and_b38.py --config TWE_config.json
+```
+
+### How it works
+The script:
+- Find all projects using the `search_term` in GRCh38 in DNAnexus.
+- Gets the related GRCh37 project for each GRCh38 project with the prefix `002_` and the suffix `_{assay}`.
+ - Within the GRCh37 and GRCh38 projects, find all files by `filename` and reads them all in with pandas.
```

Markdownlint (0.37.0)

Expected: 0; Actual: 1
Inconsistent indentation for list items at the same level

(MD005, list-indent)